fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/EsUtil.java

```diff
@@ -168,7 +171,7 @@ public static JSONObject getMappingProps(String sourceIndex, String indexMapping
         if (rootSchema == null) {
             properties = (JSONObject) mappings.get("properties");
             // Compatible es6 with no type passed in.
-            if (mappingType == null) {
+            if (mappingType == null && properties == null) {
```

Comment:
&& properties == null maybe not necessary， you check the context again

Reply:
The logic that es6 does not pass type is special, and this should not be entered in es7 or later.

Reply:
OK，we should change the location

Reply:
Okay, so I'm going to refactor this